.eslintrc.js

```diff
@@ -1,18 +1,186 @@
+require('dotenv-flow').config({
+  silent: true,
+});
+
 const CHECK_CODESTYLE = process.env.CODE_STYLE === 'true';
 const CI = Boolean(process.env.CI);
-const CHECK_SPELL_OBJECT = process.env.CHECK_SPELL_OBJECT === 'true';
 
 module.exports = {
-  extends: ['@martijnhols/eslint-config', 'plugin:react/jsx-runtime'],
-  plugins: ['wowanalyzer', 'progress'],
+  extends: [
+    'react-app',
+    // https://github.com/eslint/eslint/blob/master/conf/eslint-recommended.js
+    'eslint:recommended',
+    // https://github.com/yannickcr/eslint-plugin-react/blob/master/index.js#L115
+    'plugin:react/recommended',
+    // https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/src/configs/eslint-recommended.ts
+    'plugin:@typescript-eslint/eslint-recommended',
+    // https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/src/configs/recommended.json
+    'plugin:@typescript-eslint/recommended',
+    // We enable the prettier plugin even if ENABLE_PRETTIER is false since it
+    // also disables rules that would conflict with Prettier. We need these
+    // overrides even if we're ignoring Prettier rule problems.
+    // https://prettier.io/docs/en/integrating-with-linters.html#recommended-configuration
+    'plugin:prettier/recommended',
+    // See https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore for
+    // reasons why.
+    // https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore/blob/master/lib/rules/all.js
+    'plugin:you-dont-need-lodash-underscore/compatible',
+    'plugin:react/jsx-runtime',
+  ],
+  plugins: ['wowanalyzer', 'progress', 'no-only-tests'],
   rules: {
-    'progress/activate': process.stdout.isTTY ? 1 : 0,
+    'progress/activate': 1,
```

Comment:
don't want this I don't think? it means any tool calling `eslint` for JSON output breaks since progress data gets dumped

Reply:
yep, will get that fixed